lm_eval/tasks/__init__.py

```diff
@@ -15,6 +15,8 @@
 
 import logging
 
+from .squadv2.squad import SQuAD2
```

Comment:
Let's add a comment on the line above this import saying

`# import python tasks` ?